scripts/typemock/gulpfile.js

```diff
@@ -0,0 +1,28 @@
+const gulp = require("gulp");

```

Comment:
Is it worth considering making this a `Gulpfile.ts` just like our root one?

Reply:
I'd rather not change the extension. TypeMock was originally a garage/hackathon project I put together a few years back and eventually I'd like to see about publishing it as a standalone package. Since the gulpfile is so simple I don't think its worth adding the ts-node dependency and overhead to support a .ts extension.